fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -2524,6 +2524,8 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
                                   SimVar.SetSimVarValue('L:AIRLINER_DECISION_HEIGHT', 'feet', -1);
                                 } else if (v === null) {
                                   SimVar.SetSimVarValue('L:AIRLINER_DECISION_HEIGHT', 'feet', -2);
+                                } else if (v > 1) {
+                                  SimVar.SetSimVarValue('L:AIRLINER_DECISION_HEIGHT', 'feet', -1);
```

Comment:
This does not do what was intended at all, and will actually totally break the minima.

Reply:
Agree, we must not merge the PR in the current state